pyro/poutine/handlers.py

```diff
@@ -354,16 +354,36 @@ def mask(fn=None, mask=None):
     return msngr(fn) if fn is not None else msngr
 
 
-def indep(fn=None, name=None, size=None, dim=None):
+def indep(fn=None, name=None, size=None, dim=None, sites=None):
```

Comment:
Should we rename this `plate` and add `indep` as an alias with a `DeprecationWarning`?